### PR TITLE
mdf2iso: init at 0.3.1

### PR DESCRIPTION
**Pull request by oxij**
###### Things done
- [X] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [X] NixOS
  - [ ] OS X
  - [ ] Linux
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by joachifm** [comment]
I'm getting 
> Initialized empty Git repository in /nix/store/dpndx513xxcwh5acmm0aq05ayi24lfqy-mdf2iso-5a8acaf/.git/
fatal: repository 'https://anonscm.debian.org/collab-maint/mdf2iso.git/' not found

Travis appears to experience the same problem.

File: pkgs/tools/cd-dvd/mdf2iso/default.nix [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  name = "mdf2iso-${version}";
+  version = "0.3.1";
+
+  src = fetchgit {
+    url    = https://anonscm.debian.org/collab-maint/mdf2iso.git;
+    rev    = "5a8acaf3645bff863f9f16ea1d3632c312f01523";
+    sha256 = "0f2jx8dg1sxc8y0sisqhqsqg7pj1j84fp08nahp0lfcq522pqbhl";
+  };
+
+  meta = {
```
Comment (joachifm) [inline pkgs/tools/cd-dvd/mdf2iso/default.nix]: Please consider adding yourself as the maintainer.

**Comment by oxij** [comment]
Fixed.

**Comment by joachifm** [comment]
Thank you

**Pull request merged**